src/plugins/intel_cpu/tests/functional/shared_tests_instances/single_layer_tests/non_max_suppression.cpp

```diff
@@ -4,31 +4,30 @@
 
 #include <vector>
 
-#include "single_layer_tests/non_max_suppression.hpp"
+#include "single_op_tests/non_max_suppression.hpp"
 #include "common_test_utils/test_constants.hpp"
 
-using namespace LayerTestsDefinitions;
-using namespace InferenceEngine;
-using namespace ngraph;
+namespace {
+using ov::test::NmsLayerTest;
 
-const std::vector<InputShapeParams> inShapeParams = {
-    InputShapeParams{3, 100, 5},
-    InputShapeParams{1, 10, 50},
-    InputShapeParams{2, 50, 50}
+const std::vector<ov::test::InputShapeParams> inShapeParams = {
+    {3, 100, 5},
+    {1, 10, 50},
+    {2, 50, 50}
 };
 
 const std::vector<int32_t> maxOutBoxPerClass = {5, 20};
 const std::vector<float> threshold = {0.3f, 0.7f};
 const std::vector<float> sigmaThreshold = {0.0f, 0.5f};
-const std::vector<op::v5::NonMaxSuppression::BoxEncodingType> encodType = {op::v5::NonMaxSuppression::BoxEncodingType::CENTER,
-                                                                           op::v5::NonMaxSuppression::BoxEncodingType::CORNER};
+const std::vector<ov::op::v5::NonMaxSuppression::BoxEncodingType> encodType = {ov::op::v5::NonMaxSuppression::BoxEncodingType::CENTER,
+                                                                               ov::op::v5::NonMaxSuppression::BoxEncodingType::CORNER};
 const std::vector<bool> sortResDesc = {true, false};
-const std::vector<element::Type> outType = {element::i32, element::i64};
+const std::vector<ov::element::Type> outType = {ov::element::i32, ov::element::i64};
 
 const auto nmsParams = ::testing::Combine(::testing::ValuesIn(inShapeParams),
-                                          ::testing::Combine(::testing::Values(Precision::FP32),
-                                                             ::testing::Values(Precision::I32),
-                                                             ::testing::Values(Precision::FP32)),
+                                          ::testing::Combine(::testing::Values(ov::element::f32),
+                                                             ::testing::Values(ov::element::i32),
+                                                             ::testing::Values(ov::element::f32)),
```

Comment:
duplicate to be removed

Reply:
this is not a duplication. testing::Combine glues some values into tuple